super-size/many-extensions/pom.xml

```diff
@@ -69,10 +69,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-narayana-jta</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-spring-web</artifactId>
-        </dependency>
+        <!-- https://github.com/quarkusio/quarkus/issues/19190 -->
```

Comment:
How is spring-web related to `Multiple mechanisms present that use the same credential transport HttpCredentialTransport" Error` ?

Reply:
Is not really related, was a mistake. The commitment that pushes these changes is this one: 
https://github.com/quarkusio/quarkus/commit/149d5381208fa731914a1de2ab8fc868934cfdd1